packages/coreutils/src/pageconfig.ts

```diff
@@ -44,7 +44,7 @@ namespace PageConfig {
   export
   function getOption(name: string): string {
     if (configData) {
-      return configData[name] || '';
+      return configData[name] || Private.getBodyData(name);
```

Comment:
This behavior is documented on several downstream uses, but should probably be documented on this class or method as well.